apps/ex_nvr_web/lib/ex_nvr_web/components/layouts/app.html.heex

```diff
@@ -142,6 +142,23 @@
               <span class="flex-1 ml-3 whitespace-nowrap">Onvif Discovery</span>
             </.link>
           </li>
+          <li :if={@current_user && @current_user.role == :admin}>
+            <.link
+              href={~p"/users"}
+              class="flex items-center p-2 text-gray-900 rounded-lg dark:text-white hover:bg-gray-100 dark:hover:bg-gray-700"
+            >
+              <svg
```

Comment:
Try to use the `<.icon>` tag if the icon features in [hereicons](https://heroicons.com/)